_posts/2020-02-24-discerning-and-maintaining-purity.html

```diff
@@ -296,4 +296,22 @@ <h3 id="67101ecee7b94a849b24126ac01851fe">
 		</ul>
 		If you want to harvest the benefits of functional programming in a mainstream language, you must look out for such pitfalls. There's no tooling to assist you.
 	</p>
-</div>
\ No newline at end of file
+</div>
+<div id="comments">
+	<hr>
+	<h2 id="comments-header">
+		Comments
+	</h2>
+	<div class="comment" id="1925963B8A194CF9BC44E0EB59E92C9C">
+		<div class="comment-author"><a href="http://criticalsoftwareblog.com">Yacoub Massad</a></div>
+		<div class="comment-content">
+			<p>
+				You might be interested in taking a look at [PurityAnalyzer](https://github.com/ymassad/PurityAnalyzer). An open source roslyn-based analyzer for C# that I started developing to help maintain pure C# code.
```

Comment:
@ymassad this links isn't valid

Reply:
The link itself is valid. I thought I could use Markdown. Will change to an an element.

Reply:
@ymassad yes, I meant the link itself is not rendered 🙂

Reply:
I have edited it to fix it. Thanks.